From 03fc6349b1e5f715b05c67976a7b33f4703a89b4 Mon Sep 17 00:00:00 2001
From: Ryan Brady <rbrady@redhat.com>
Date: Wed, 19 Nov 2014 17:41:14 -0500
Subject: [PATCH] Update RHEL Registration

This patch updates the 00-rhel-registration script to not
not fail a stack when metadata isn't found.

Change-Id: Ie305fed79e4baadf1a03c4a3d06a23cf36e92f77
---
 .../rhel-common/os-refresh-config/pre-configure.d/06-rhel-registration  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/rhel-common/os-refresh-config/pre-configure.d/06-rhel-registration b/elements/rhel-common/os-refresh-config/pre-configure.d/06-rhel-registration
index 631f0c6..f85569f 100755
--- a/elements/rhel-common/os-refresh-config/pre-configure.d/06-rhel-registration
+++ b/elements/rhel-common/os-refresh-config/pre-configure.d/06-rhel-registration
@@ -115,5 +115,5 @@ case "${REG_METHOD:-}" in
         ;;
     *)
         echo "WARNING: only 'portal' and 'satellite' are valid values for REG_METHOD."
-        exit 1
+        exit 0
 esac
